pkg/logger/file.go

```diff
@@ -27,30 +27,34 @@ import (
 type FileLogger struct {
 	Logger   *log.Logger
 	LogLevel LogLevel
+	out      *os.File
 }
 
 // NewFileLogger ...
 func NewFileLogger(name string, file string) (logger Logger, out *os.File, err error) {
-	if out, err = setup(file); err != nil {
+	out, err = setup(file)
+	if err != nil {
 		return nil, nil, err
 	}
 	logger = &FileLogger{
+		out:      out,
 		Logger:   log.New(out, name, log.LstdFlags),
-		LogLevel: logLevelFromEnvironment(),
+		LogLevel: LogLevelFromEnvironment(),
```

Comment:
isn't the logLevel already specified in `logger.NewLogger(&logger.Options{...})`

Reply:
the plan is to resolve this in another PR for unified logger, the factory methods do not accept options as a parameter, and hence why this is initialised as such. Tracking it here

https://github.com/codenotary/immudb-issues/issues/85